docs/guides/hedgehog-engine/unleashed/levels/glvl.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Sonic GLvl setup tutorial for Sonic Unleashed
+---
+
+# How to use Sonic GLvl with Sonic Unleashed
+
+!!! note
+    This guide covers how to get unleashed levels working in order to edit them, this guide does not cover how to use GLvl. its recommended you also read the [Generations GLvl guide.](/guides/hedgehog-engine/blueblur/levels/glvl.md){ target="_blank"}
+
+
+    
+!!! info
+
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+    
+    - [Unleashed Mod Toolbox](/tools/hedgehog-engine/unleashed/files){ target="_blank"}  
+
+    - [Unleashed Object Template](assets/glvl/UnleashedObjectTemplate.zip)
+
+    - Xbox 360 SDK, find this yourself
+
+
+
+
+
+
+
+#### Part 1 (Preparing .ar Files)
```

Comment:
You don't need to specify "Part {number}", as the person is already reading the page from top to bottom. Just write the chapter titles :)